src/style.css

```
@@ -1 +1,11 @@
-/* styles go here */
+.form_fieldset {
+  margin-bottom: 20px;
+}
+
+.form__input {
+  margin-bottom: 10px;
+}
+
+.form__input:last-child {
+  margin-bottom: 0;
+}
```

You can combine both rules
```suggestion
.form__input:not(:last-child) {
  margin-bottom: 10px;
}
```